py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2626,3 +2626,20 @@ def aten_ops_pdist(
         args[0],
         args_bounds_check(args, 1, 2),
     )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.scalar_tensor.default)
+def aten_ops_scalar_tensor(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.unary.scalar_tensor(
+        ctx,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+    )
```

Comment:
Consider adding support for the `dtype` keyword argument, which we can support through the `get_trt_tensor` API.